test/e2e/scenarios/scenario1.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
+#marks:
+#  - slow
+#  - blocked
+customResourceReference:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
```

Comment:
where does `$RANDOM_SUFFIX` come from?

Reply:
Method:
`load_scenario(scenario_file: Path, resource_directory: Path = None, replacements: dict = {}) -> model.Scenario` 

sets `RANDOM_SUFFIX` in `replacements` dictionary before loading the scenario file. 

Each scenario gets different random suffix.